src/launcher.ts

```diff
@@ -28,9 +31,34 @@ export default class BrowserstackLauncherService implements Services.ServiceInst
     onPrepare (config?: Options.Testrunner, capabilities?: Capabilities.RemoteCapabilities) {
         if (Array.isArray(capabilities)) {
             capabilities.forEach((capability: Capabilities.DesiredCapabilities | any) => {
-                const wdioServiceVersion = process.env.npm_package_version;
+                const wdioServiceVersion = version;
+                let webdriverIOVersion: any = undefined;
+                const packageFile: any = process.env.npm_package_json;
```

Comment:
`const packageFile: any = process.env.npm_package_json;`
Wont this point to package.json of the repo instead of lib?

Reply:
yes, intended. Want to get the webdriver version used in repo.